helm/Chart.yaml

```diff
@@ -29,4 +29,4 @@ dependencies:
   - name: kafka-topic-creator
     condition: kafka-topic-creator.enabled
     repository: "https://storage.googleapis.com/hypertrace-helm-charts"
-    version: 0.2.0
+    version: 0.2.x
```

Comment:
What 'x' represents here ?

Reply:
it will pull the latest version starting with 0.2.

Reply:
I'm not sure if that's a good idea as compared to pinning to a specific version number. I know we are probably trying to avoid constant upgrades to this but I wonder if it's worth sacrificing precision.

Reply:
This will avoiding a number of repos every time minor version is released. If there is any breaking change in `kafka-topic-creator` helm chart then we should increase major version instead.